distribution/docs/src/main/resources/content/_managing/_running/ddf-service.adoc

```diff
@@ -64,7 +64,11 @@ LimitNOFILE=6815744
 Install the service and start it when the system boots, use `systemctl` From an OS console, execute:
 +
 `root${at-symbol}localhost# systemctl enable ${home_directory}/bin/${branding-lowercase}.service`
-
++
+[WARNING]
+====
+Some *NIX operating systems may require additional configuration if the .service script is located outside of the root filesystem, or is symlinked to a .service file in another partition.
```

Comment:
❓ should more be added on what is the additional configuration?

Reply:
I'm hesitant to provide specifics in the documentation, because it varies by OS and installation, so the specific configurations might not be consistent and will have to be continually updated to ensure we always have proper instructions. I'll reach out offline to talk about this more.

Reply:
No worries. Since this is an edge case: 👍

Reply:
👍